go/cmd/vtctldclient/command/tablets.go

```diff
@@ -421,11 +427,20 @@ func commandGetTablets(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("--shard (= %s) cannot be passed without also passing --keyspace", getTabletsOptions.Shard)
 	}
 
+	tabletType := topodatapb.TabletType_UNKNOWN // Zero Value
```

Comment:
there is a tablet-type flag you can use in `topoproto` to parse directly into the correct type